uploadFile.js

```diff
@@ -13,18 +13,19 @@ const API_URL = 'https://api.github.com/gists';
 async function uploadFile(contents, gist) {
   if (!gist || !contents) {
```

Comment:
check for these outside of the uploadFile function, i.e. if `process.args[x]` is undefined, etc

Reply:
done